rbtcwrapperproxy/interfaces/ILiquidityPoolV1Converter.sol

```diff
@@ -12,7 +12,7 @@ import "./IConverterAnchor.sol";
   * Even though classic pools can have many reserves, the most common configuration of
   * the pool has 2 reserves with 50%/50% weights.
 */
-interface ILiquidityPoolV1Converter {
+contract ILiquidityPoolV1Converter {

```

Comment:
please keep your old `interface`. create an abstract `contract` for data in a separate file if you need one.